src/scenes/AcceleratorRouter/MatrixView/ColumnsModal.tsx

```diff
@@ -282,9 +282,14 @@ export default function ColumnsModal(props: ColumnsModalProps): JSX.Element {
     [table]
   );
 
+  const baseColumns = useMemo(
+    () => ['projectName', 'participantCohortPhase', 'elegibleLand', 'countryName', 'projectLead'],
+    []
+  );
```

Comment:
Since this is static, could also move to above the component
```ts
const BASE_COLUMNS = ['projectName', 'participantCohortPhase', 'elegibleLand', 'countryName', 'projectLead'];
```